registration-processor/registration-processor-packet-manager/src/main/java/io/mosip/registration/processor/packet/manager/idreposervice/IdrepoDraftService.java

```diff
@@ -62,24 +64,32 @@ public ResponseDTO idrepoGetDraft(String id) throws ApisResourceAccessException
         regProcLogger.debug("idrepoGetDraft entry " + id);
         IdResponseDTO idResponseDTO = (IdResponseDTO) registrationProcessorRestClientService.getApi(
                 ApiName.IDREPOGETDRAFT, Lists.newArrayList(id), Lists.emptyList(), null, IdResponseDTO.class);
+        if (idResponseDTO.getErrors() != null && !idResponseDTO.getErrors().isEmpty()) {
+            ErrorDTO error = idResponseDTO.getErrors().get(0);
+            regProcLogger.error("Error occured while getting draft for id : " + id, error.toString());
+        }

```

Comment:
Need to throw the IDRepDraftException from here.

Reply:
Added Exception.